ibm/structures.go

```diff
@@ -137,3 +139,64 @@ func flattenServiceInstanceCredentials(keys []mccpv2.ServiceKeyFields) []interfa
 	}
 	return out
 }
+
+func flattenIAMPolicyResource(list []iampapv1.Resources, iamClient iampapv1.IAMPAPAPI) []map[string]interface{} {
+	result := make([]map[string]interface{}, 0, len(list))
+	for _, i := range list {
+		name := i.ServiceName
+		var serviceName string
+		if name != "" {
+			serviceName, _ = iamClient.IAMService().GetServiceDispalyName(name)
+		} else {
+			serviceName = "All Identity and Access enbled services"
+		}
```

Comment:
If we are assigning `All Identity and Access enbled services`.(Also note that  that spelling is incorrect) then we should define it as default value in the schema